lib/ae_mdw/db/node_store.ex

```diff
@@ -0,0 +1,85 @@
+defmodule AeMdw.Db.NodeStore do
+  @moduledoc """
+  Store implementation with operations reading from the node tables directly.
+  """
+
+  alias AeMdw.Db.Model
+
+  require Model
+
+  @derive AeMdw.Db.Store
+  defstruct []
+
+  @typep key() :: term()
+  @typep record() :: tuple()
+  @typep table() :: table()
+  @opaque t() :: %__MODULE__{}
+
+  @spec new() :: t()
+  def new, do: %__MODULE__{}
+
+  @spec put(t(), table(), record()) :: t()
+  def put(_store, _table, _record) do
+    raise "Not implemented"
```

Comment:
How are values getting written into ETS?

Reply:
The records are controlled by the node, the mdw just reads the table. I don't think we need to modify data there

Reply:
But values are being read from a table called Model.Mempool in ETS. Is that what the node is using?

Reply:
That's just a handle for the table, the actual table being read from is :mempool. The read functions in the store resolve Model.Mempool to :mempool

Reply:
I see now, Model.Mempool is being converted to :mempool using Model.record/1. Thanks